src/pudl/constants.py

```diff
@@ -2641,6 +2665,13 @@
         'liquefied_natural_gas_storage': pd.BooleanDtype(),
         'longitude': float,
         'mercury_content_ppm': float,
+        'merge_address': pd.StringDtype(),  # Added by AES for Mergers table
+        'merge_city': pd.StringDtype(),  # Added by AES for Mergers table
+        'merge_company': pd.StringDtype(),  # Added by AES for Mergers table
+        'merge_date': 'datetime64[ns]',  # Added by AES for Mergers table
+        'merge_state': pd.StringDtype(),  # Added by AES for Mergers table
+        'merge_zip_4': pd.StringDtype(),  # Added by AES for Mergers table
+        # 'merge_zip_5': pd.StringDtype(),  # Added by AES for Mergers tables
```

Comment:
DId you intend for this to remain commented out?

Reply:
no, thanks for catching that